src/app/shared/notifications/notifications-board/notifications-board.component.html

```diff
@@ -1,7 +1,10 @@
-<div class="notifications-wrapper position-fixed" [ngClass]="position">
+<div class="notifications-wrapper position-fixed"
+     [ngClass]="position"
+     (mouseenter)="this.isPaused$.next(true);"
+     (mouseleave)="this.isPaused$.next(false);">
```

Comment:
Could we add some basic specs to verify that pausing occurs on mouse enter & stops on leave?